# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../enblend/max.patch
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- enblend-3.0/src/anneal.h_orig	2007-08-24 21:50:17.000000000 +0200
+++ enblend-3.0/src/anneal.h	2007-08-24 21:55:09.000000000 +0200
@@ -678,7 +678,7 @@
 
             localK = stateSpace->size();
             if (localK < 2) convergedPoints[index] = true;
-            kMax = std::max(kMax, stateProbabilities->size());
+            kMax = std::max(kMax, (unsigned int)(stateProbabilities->size()));
 
         }
 
